app/api/auth/router.py

```diff
@@ -339,6 +339,19 @@ async def first_setup(
                     },
                     "objectSid": 513,
                 },
+                {
+                    "name": "read only",
+                    "object_class": "group",
+                    "attributes": {
+                        "objectClass": ["top", "posixGroup"],
+                        "groupType": ["-2147483646"],
+                        "instanceType": ["4"],
+                        "sAMAccountName": ["domain users"],
+                        "sAMAccountType": ["268435456"],
+                        "gidNumber": ["514"],
+                    },
+                    "objectSid": 514,
```

Comment:
DOMAIN_GUESTS

S-1-5-21-<domain>-514

A global group that has only one member, which is the built-in Guest account of the domain.